pages/sessions.jsx

```diff
@@ -232,7 +227,7 @@ var SpacesInfo = [
 
 var SessionsPage = React.createClass({
   spaces: SpacesInfo,
-  componentWillMount: function() {
+  render: function() {
```

Comment:
it raised errors if we attempted to use <Link> in <SpacePathwayProfile>
```
Failed Context Types: Required context `router` was not specified in `Link`.
```

I tried to look for causes of this problem and this was the only fix I found. It doesn't seem like our router was set up wrong... or maybe I'm wrong?